src/lib/index.js

```diff
@@ -1145,6 +1176,14 @@ function templateFormatString(string, labels, d3locale) {
             }
         }
 
+        // Apply mult/div operation (if applicable)
+        if(parsedOp) {
```

Comment:
Let's be more strict here e.g.:
```js
        if(parsedOp !== null) {
```
Or even better would be
```suggestion
        if(parsedOp === '*' || parsedOp === '/') {
```